misc/keybindings.clifm

```diff
@@ -125,6 +125,7 @@ toggle-disk-usage:\C-\M-i
 toggle-virtualdir-full-paths:\M-w
 trash-sel:\C-\M-t
 untrash-all:\C-\M-u
+xclip:\C-y
```

Comment:
Hey @theRoboxx. This won't work. There are four available "slots" for plugins in the keybindings file: `plugin1`, `plugin2`, `plugin3`, and `plugin4`.

To make it work, just replace this line by this one: `plugin1=\C-y`
Also, the line in the actions file must match the name used here. So that it should be: `plugin1=xclip.sh`.